### ISSUE TITLE
Example didn't work

### ISSUE THREAD

**Issue by yookoala** [hongkongkiwi/node-hongkong-weather#1]
Copied this straight from the README.md example:
```javascript
var HongKongWeather = require('hongkong-weather');

var hkWeather = HongKongWeather();

hkWeather.getForecast().then(function(forecast){
    console.log(forecast);
});
```

It fails with this error:
```
/home/user/mytest/index.js:5
hkWeather.getForecast().then(function(forecast){
         ^

TypeError: Cannot read property 'getForecast' of undefined
    at Object.<anonymous> (/home/user/mytest/index.js:5:10)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
```

**Comment by alankyshum** [comment]
TLDR; this repo's still **WIP**

the variable need to be initialised using
```javascript
var hkWeather = new HongKongWeather();
```

and that function does not even exists (check `index.js`), but only those here:
```
getCurrent (only implemented function)
getWeatherWarnings
getShortForecast
getLongForecast
getRadiationLevels
getLocalEarthqakes
getWorldEarthqakes
getTidalData
getSunriseData
```

However, there's certainly a lack of examples of how this repo works.

**Comment by hongkongkiwi** [comment]
Do you guys need anymore examples? I've just come back to this after a while, but happy to add any features you need.
